conans/client/cmake.py

```diff
@@ -40,7 +40,8 @@ def _generator(self):
                         '10': '10 2010',
                         '11': '11 2012',
                         '12': '12 2013',
-                        '14': '14 2015'}
+                        '14': '14 2015',
+                        '15': '15 2017'}
```

Comment:
Thats very interesting to know, this would be something to define in ``conan_info()`` (today release rename it to ``package_id()``). Being compatible doesn't mean that the resulting binary would be the same, typically modern versions are able to do better optimizations, for example. If a package creator wants to use the same binary for both 14 and 14.1 toolsets (or vs14 and vs15), they can easily define it in ``package_id()``. We will be releasing new documentation for it today too.